src/web/server/document.tsx

```diff
@@ -145,6 +155,7 @@ export const document = ({ data }: Props) => {
         title,
         description: CAPI.trailText,
         windowGuardian,
+        gaPath,
```

Comment:
we put this on windowGuardian so that we can construct the correct paths when we do want to load GA scripts